jormungandr/src/blockchain/chain.rs

```diff
@@ -298,6 +298,13 @@ impl Blockchain {
         &mut self.branches
     }
 
+    pub async fn gc(&self, tip: Arc<Ref>) -> Result<()> {
+        let depth = tip.epoch_ledger_parameters().epoch_stability_depth;
+        self.ledgers.gc(depth).await;
+        self.storage.gc(depth, tip.hash().as_ref())?;
```

Comment:
What's the effect if the storage GC operation fails? Is it bad to have storage branches that are not attested in the ledger multiverse? May need to swap these two.